eradiate/radprops/rad_profile.py

```diff
@@ -579,9 +595,9 @@ def to_dataset(self, spectral_ctx=None):
             Radiative properties dataset.
         """
         profile = self.eval_thermoprops_profile()
-
+        wavelength = spectral_ctx.wavelength
```

Comment:
If you don't reuse that variable, there is no point creating it. If you do that for the sake of ensure that the spectral context has a `wavelength` member, no worries: it must have one (and will raise if querying it makes no sense).
```suggestion
```